src/sfizz/Voice.h

```diff
@@ -284,7 +288,15 @@ class Voice {
     ADSREnvelope<float> egEnvelope;
     float bendStepFactor { centsFactor(1) };
 
-    WavetableOscillator waveOscillator;
+    static constexpr unsigned maxWaveOscillators = Default::oscillatorMultiRange.getEnd();
```

Comment:
An alternative for this one would be to put it in config and initialized the `Default::oscillatorMultiRange` as `{ 1, config::maxWaveOscillators }`